src/Transport/Creation/AzureServiceBusQueueCreator.cs

```diff
@@ -40,6 +43,28 @@ public AzureServiceBusQueueCreator(ReadOnlySettings settings)
             }
         }
 
+        void TryGetSystemQueuessAddresses()
```

Comment:
I'm starting to doubt that we need this since I think that all the queues passed in the "sending" collection should be consider as shared/system queues? (Audit+Forwarding+Error) 

In short: Treat all queues in https://github.com/Particular/NServiceBus/blob/release-6.0.0/src/NServiceBus.Core/Transports/QueueBindings.cs#L18 as system/shared

Side note: I don't think forwarding should be there, I'll raise a core issue for that

@Particular/nservicebus-maintainers what do you all think?